icons/deutschebahn.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Deutsche Bahn icon</title><path d="M21.6 3.6H2.4C1.1 3.6 0 4.7 0 6v12c0 1.3 1.1 2.4 2.4 2.4h19.2c1.3 0 2.4-1.1 2.4-2.4V6c0-1.3-1.1-2.4-2.4-2.4zm.6 14.4c0 .4-.3.7-.6.7H2.4c-.4 0-.6-.3-.6-.7V6c0-.4.3-.7.6-.7h19.2c.4 0 .6.3.6.7v12zM7.3 6.5h-4v11h4c3.3 0 4.4-2.4 4.4-5.5.1-3.7-1.5-5.5-4.4-5.5zm-.4 9.2H6V8.2h.5c2.4 0 2.6 1.7 2.6 3.9 0 2.5-.8 3.6-2.2 3.6zm11.8-4c1.1-.3 1.9-1.4 1.9-2.6 0-.2 0-2.7-3.1-2.7h-4.6v11h3.9c1 0 4.2 0 4.2-3.1 0-.7-.3-2.1-2.3-2.6zm-2.3-3.5c1.2 0 1.7.4 1.7 1.3 0 .7-.5 1.3-1.4 1.3h-1V8.2h.7zm.2 7.5h-1.1v-2.8h1.2c1.4 0 1.7.8 1.7 1.4 0 1.4-1.4 1.4-1.8 1.4z"/></svg>
```

Comment:
It looks a little bit different than the original. It might be during the optimization process. You can try with a precision level of 3 which is good for most of the icon.
![image](https://user-images.githubusercontent.com/12817388/100802553-ad28c100-33f7-11eb-91f0-f61ef70b72ad.png)

Reply:
I used level 3 (and it seems same to me).
Maybe, should I use another level instead of 3?

Reply:
Yes, usually this is a result of optimizing the SVG. You can try a precision of 4, that is usually good enough, lastly you can resort to a precision of 5.

Regarding @service-paradis comment, judging by the contents of the SVG's path, it looks like you used a precision of 1. Maybe you're editor already did some optimization?

You can try to compare the version before you optimize to the one after you optimize to be sure the optimization didn't mess something up.

Reply:
Hi @FullteaOfEEIC, did you still want to look at this icon? I'm happy to commit any changes if you're no longer able to look at it.